.zuul.yaml

```diff
@@ -0,0 +1,28 @@
+- project:
+    name: kubernetes-sigs/cluster-api-provider-openstack
+    check:
+      jobs:
+        - cluster-api-provider-openstack-conformance-stable
+
+- job:
+    name: cluster-api-provider-openstack-conformance-stable
+    parent: init-test
+    timeout: 36000
+    description: |
+      Test CAPO & Kubernetes master against CAPI & CABPKG stable
+    run: .zuul/playbooks/run.yaml
+    nodeset: ubuntu-xenial-citynetwork
```

Comment:
Oh, one line is missing: 
```
    secrets:
      - citynetwork_credentials
```
example: https://github.com/theopenlab/openlab-zuul-jobs/blob/master/zuul.d/jobs.yaml#L534